src/lib/pickles_types/hlist.ml

```diff
@@ -756,7 +748,7 @@ module type Max_s = sig
 
   val n : n t
 
-  val p : (ns, n) Hlist_1(Lte).t
+  val p : (ns, n) Hlist0.H1_1(Lte).t
```

Comment:
Not sure I understand, are you suggesting to `include Hlist0` above ? (which is possible, but then we should remove `module Length = Hlist0.Length` etc.

Reply:
I suppose it's fine as it is, but it would be nice to lift the definition of `H1_1` into the spot where `Hlist_1` was defined before, so that we aren't explicitly referring to `Hlist0` here.

Reply:
Alright, let's put a `module H1_1 = Hlist0.H1_1` where `Hlist_1` was.